.env.example

```diff
@@ -18,3 +18,8 @@ export SIDEKIQ_PASSWORD=foo
 export MENDELEY_CONSUMER_KEY=foo
 export ERRBIT_API_KEY=foo
 export ERRBIT_HOST=localhost
+
+export SURVEY_EMAIL_USER=foobar
+export SURVEY_EMAIL_PASSWORD=foobar
+export SURVEY_EMAIL_ADDRESS=foobar
+export SURVEY_EMAIL_PORT=foobar
```

Comment:
I don't understand what these are for. Isn't it enough to just set the sender's email address?